### ISSUE TITLE
Concentrations North/South inverted on maps

### ISSUE THREAD

**Issue by barronh** [geoschem/gcpy#97]
## Describe the bug

I ran the examples/compare_diagnostics.ipynb, but all the surface level maps had concentrations inverted. The map outlines were correct, but the concentration data was inverted.  This is related to the imshow call on line 2585 in 0.3.1 and main.  Not sure why, but the origin parameter for imshow defaults to upper. I tried updating mpl.rcParams['image.origin'] to "lower", but I had to specify origin="lower" in the imshow call.

Likely related to versions. I was unable to install using the docs/environment.yml and had to install bit by bit, but it seems like an easy fix. I was working off the "main" branch, but also tested the 0.3.1

## To Reproduce
I encountered the issues in the notebook, but can reproduce it with the few commands below.

```
import xarray as xr
import gcpy
# Set netcdf ref data if doing nc vs nc
refdata = xr.open_dataset('/work/ROMO/global/GCv12.0.1/GC/rundirs/geosfp_4x5_tropchemref/outputs/GEOSChem.SpeciesConc.20160701_0000z.nc4')
    
# Set netcdf dev data if doing bpch vs nc
devdata = xr.open_dataset('/work/ROMO/global/GCv12.0.1/GC/rundirs/geosfp_4x5_tropchem16ww/outputs/GEOSChem.SpeciesConc.20160701_0000z.nc4')

# Make plots
gcpy.compare_single_level( refdata, 'GCC netcdf, 12.0.1 (Ref)', devdata, 'GCC netcdf, 12.0.1 (NEI16)', ilev=0, varlist=['SpeciesConc_O3'], pdfname='Test.pdf')
```

## Expected behavior
Northern Hemisphere concentrations should be over the northern hemisphere.

## Required information:
Run the following commands from an interactive Python session:

Unable to report versions
```
>>> import gcpy
>>> gcpy.show_versions()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'gcpy' has no attribute 'show_versions'
>>>
```

**Comment by WilliamDowns** [comment]
Thanks for this and apologies for this issue. This is related to Cartopy 0.18.0 (see #90) and will be fixed for the 1.0.0 release.

**Comment by barronh** [comment]
Thanks.
